.github/workflows/sanitizer.yml

```diff
@@ -50,14 +50,17 @@ jobs:
       run: |
         echo "ASAN_OPTIONS=alloc_dealloc_mismatch=0" >> $GITHUB_ENV
 
-    - name: setup clang
-      uses: egor-tensin/setup-clang@v1
-      with:
-        version: latest
-        platform: x64
+    - name: Installing build dependencies
+      run: |
+        sudo apt update
+        sudo apt install clang-19
 
     - name: configure clang
       run: |
+        sudo ln -s /usr/bin/clang-19 /usr/local/bin/clang
+        sudo ln -s /usr/bin/clang++-19 /usr/local/bin/clang++
+        sudo ln -s /usr/bin/clang /usr/local/bin/cc
+        sudo ln -s /usr/bin/clang++ /usr/local/bin/c++
```

Comment:
you should be able to just `echo "CC=/usr/bin/clang-19" >> $GITHUB_ENV` instead

Reply:
SomeI guess it doesn't matter for us, but in general, doing just that might not be enough in some cases.